### ISSUE TITLE
No diagnostic given when pointer is passed to explicit interface taking non-pointer

### ISSUE THREAD

**Issue by schweitzpgi** [flang-compiler/f18-llvm-project#1044]
```fortran
 real, pointer :: x
 interface
   subroutine extern1(r1)
   end
 end interface
 call extern1(null(x))
end
```

**Comment by klausler** [comment]
The fix for this has made its way into fir-dev; closing bug.
